docs/doing-nothing.md

```diff
@@ -16,6 +16,6 @@ If the [Fake is strict](strict-fakes.md), an unconfigured call will throw an exc
 Or, `DoesNothing` can be used to change the behavior that an already-configured call is supposed to have. For example, if a call is [set to throw an exception](throwing-exceptions.md), that can be overridden. For more on this kind of thing, see how to [override the behavior for a call](changing-behavior-between-calls.md#overriding-the-behavior-for-a-call).
 
 **Note that `DoesNothing` is only applicable when configuring members that have a
-`void` return.** To override behaviour on already-configured (or strict) members
-that return values, you must instead
-[configure a preferred return value](specifying-return-values.md).
\ No newline at end of file
+`void` return (or `Task`, which is the async equivalent of `void`).** To override
+behavior on already-configured (or strict) members that return values, you must
+instead [configure a preferred return value](specifying-return-values.md).

```

Comment:
Note: I changed "behaviour" to "behavior", since we had settled on using the US spelling.